### PR TITLE
add askJupyter option

### LINKED ISSUE #16: When Python is not available, don't run the sys.prefix check
Running nteract on a box without Python will fail on startup when it's ok for Python not to be on the system.

### PR DESCRIPTION
**Pull request by minrk**
deprecates `withSysPrefix: true` in favor of the good-enough guess based on PATH.

also closes #16

if going with this, can skip #20

### CODE REVIEW

File: index.js
Comment (rgbkrk) [inline index.js:88]: Neat!

**Review by rgbkrk — COMMENTED** [review]
I would still want to keep sys prefix available since we can't guarantee jupyter is on the system (nteract should be able to install and launch if jupyter is not available). However, I'd love to see this option so we can use jupyter if it is available.

**Comment by minrk** [comment]
> I would still want to keep sys prefix available since we can't guarantee jupyter is on the system

sys-prefix via `guessSysPrefix` will still be available by default.

File: index.js
Comment (rgbkrk) [inline index.js:4]: Good catch here, thank you. This was copied over from `ipython-paths`.

**Review by rgbkrk — APPROVED** [review]

**Comment by rgbkrk** [comment]
After rebasing, I'll go ahead and merge this.

**Comment by minrk** [comment]
rebased

File: index.js
Comment (lgeiger) [inline index.js]: ☝️

File: index.js
Comment (lgeiger) [inline index.js]: 👆

**Review by lgeiger — COMMENTED** [review]
Oops, some of the merge conflicts slipped through. I marked them in the above.

File: index.js
Comment (minrk) [reply index.js]: whoops! Fixed.

**Comment by rgbkrk** [comment]
Finally shipped 1.1.0 with this, thanks @minrk. You're also on npm as a maintainer for this package.

**Comment by minrk** [comment]
cool, thanks!

**Pull request merged**
